composer.json

```diff
@@ -45,26 +45,26 @@
         "phpunit/phpunit": "^6.1",
         "psr/log": "^1.0",
         "sensio/framework-extra-bundle": "^3.0.11 || ^4.0",
-        "symfony/asset": "^3.3 || ^4.0",
-        "symfony/cache": "^3.3 || ^4.0",
-        "symfony/config": "^3.3 || ^4.0",
-        "symfony/console": "^3.3 || ^4.0",
+        "symfony/asset": "^3.4 || ^4.0",
+        "symfony/cache": "^3.4 || ^4.0",
+        "symfony/config": "^3.4 || ^4.0",
+        "symfony/console": "^3.4 || ^4.0",
         "symfony/debug": "^2.8 || ^3.0 || ^4.0",
-        "symfony/dependency-injection": "^3.3 || ^4.0",
+        "symfony/dependency-injection": "^3.4 || ^4.0",
         "symfony/doctrine-bridge": "^2.8.12 || ^3.0 || ^4.0",
-        "symfony/event-dispatcher": "^3.3 || ^4.0",
+        "symfony/event-dispatcher": "^3.4 || ^4.0",
         "symfony/expression-language": "^2.8 || ^3.0 || ^4.0",
-        "symfony/finder": "^3.3 || ^4.0",
-        "symfony/form": "^3.3 || ^4.0",
-        "symfony/framework-bundle": "^3.3 || ^4.0",
-        "symfony/phpunit-bridge": "^3.3 || ^4.0",
-        "symfony/routing": "^3.3 || ^4.0",
-        "symfony/security": "^3.0 || ^4.0",
-        "symfony/security-bundle": "^3.0 || ^4.0",
-        "symfony/twig-bundle": "^3.1 || ^4.0",
-        "symfony/validator": "^3.3 || ^4.0",
-        "symfony/web-profiler-bundle": "^3.3 || ^4.0",
-        "symfony/yaml": "^3.3 || ^4.0",
+        "symfony/finder": "^3.4 || ^4.0",
+        "symfony/form": "^3.4 || ^4.0",
+        "symfony/framework-bundle": "^3.4 || ^4.0",
+        "symfony/phpunit-bridge": "^3.4 || ^4.0",
+        "symfony/routing": "^3.4 || ^4.0",
+        "symfony/security": "^3.4 || ^4.0",
+        "symfony/security-bundle": "^3.4 || ^4.0",
+        "symfony/twig-bundle": "^3.4 || ^4.0",
+        "symfony/validator": "^3.4 || ^4.0",
+        "symfony/web-profiler-bundle": "^3.4 || ^4.0",
+        "symfony/yaml": "^3.4 || ^4.0",
         "webonyx/graphql-php": "^0.10.2"
     },
     "conflict": {
```

Comment:
It still needs to be there, since we're not requiring `symfony/dependency-injection`.

Reply:
But bumped to `3.4`.

Reply:
With the current requirements allowing symfony ~2.7 and ~3.0:

```
"symfony/http-kernel": "^2.7 || ^3.0 || ^4.0",
"symfony/property-access": "^2.7 || ^3.0 || ^4.0",
```

There is still the same conflict when trying to install the core without the official api-platform/api-platform.

It's useless to bump it, the conflict was real only for <3.3
I don't know what is the best solution if api-platform drop support for 3.3

Reply:
It will not work with version >3.4, so we need to bump the version/

Reply:
Just FYI, the conflict was added only on symfony/dependency-injection because the bundle was using a specific feature added in symfony/dependency-injection 3.3.
I haven't really followed api-platform latest changes but if you want to drop support for symfony <3.4, bumping this conflict will be easier for everyone :)

Reply:
Same case here, we introduce a specific feature added in in `symfony/dependency-injection` 3.4. 😉

Reply:
thanks guys <3